api/src/main/java/org/openmrs/module/fhir2/api/FhirLocationService.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api;
+
+import org.hl7.fhir.r4.model.Location;
+
+import javax.validation.constraints.NotNull;
+
+public interface FhirLocationService {
+
+    Location getLocationByUuid(@NotNull String uuid);
+} 
```

Comment:
I'd prefer it if we had blank newlines at the end of files. It prevents some issues that can arise in git.